city_scrapers/spiders/example.py

```diff
@@ -0,0 +1,231 @@
+        pattern += r" - (?P<time>(\d{1,2}:\d{2}) (PM|AM))"
+        rm = re.search(pattern, date)
+
+        if rm is not None:
+            day = rm.group("day")
+            time = rm.group("time")
+            dt = day + " " + time
+            end = datetime.strptime(dt.strip(), "%m/%d/%y %H:%M %p")
+            return end
+        else:
+            return None
+
+    def _parse_all_day(self, response):
+        """Parse or generate all-day status. Defaults to False."""
+        return False
+
+    def _parse_location(self, response):
+        """Parse or generate location."""
+        location = response.css("div.col-md-4 div.content-block p *::text").getall()
+        temp = []
+        for item in location:
+            item = item.replace("\n", "")
+            if item != "":
+                temp.append(item)
+        location = temp
+        i, location_index, sponsor_index = 0, 0, 0
+        while i < len(location):
+            if "location" in location[i].lower():
+                location_index = i
+            if "sponsor" in location[i].lower():
+                sponsor_index = i
+                break
+            i += 1
+
+        if location_index + 1 < len(location) and sponsor_index < len(location):
+            name = location[location_index + 1]
+            address = []
+            for j in range(location_index + 2, sponsor_index):
+                address.append(location[j])
+            address = (
+                " ".join(address).replace("Directions to this address", "").strip()
+            )
+        else:
+            name = ""
+            address = ""
+
+        return {
+            "address": address,
+            "name": name,
+        }
+
+    def _parse_links(self, response):
+        """Parse or generate links."""
+        rows = response.css("table.data tr")
+        for row in rows:
+
+            temp_links = []
+            link = row.css("a::attr(href)").getall()
+            description = row.css("td *::text").getall()
+            description = "".join(description).replace("\n", " ")
+
+            pattern_mmddyy = r"(?P<date>(\d{1,2}-\d{1,2}-\d{2}))"
+            pattern_mmddyyyy = r"(?P<date>(\d{1,2}-\d{1,2}-\d{4}))"
+            pattern_monthddyyyy = r"(?P<date>([A-Z]* \d{1,2}, \d{4}))"
+
+            rm_mmddyy = re.search(pattern_mmddyy, description)
+            rm_mmddyyyy = re.search(pattern_mmddyyyy, description)
+            rm_monthddyyyy = re.search(pattern_monthddyyyy, description)
+
+            dt = None
+            if rm_mmddyy is not None:
+                date = rm_mmddyy.group("date")
+                dt = datetime.strptime(date, "%m-%d-%y")
+            if rm_mmddyyyy is not None:
+                date = rm_mmddyyyy.group("date")
+                dt = datetime.strptime(date, "%m-%d-%Y")
+            if rm_monthddyyyy is not None:
+                date = rm_monthddyyyy.group("date")
+                dt = datetime.strptime(date, "%b %d, %Y")
```

Comment:
None of these regex patterns match the way the date is formatted for the Public Service Board meeting materials. 
```
pattern = r"(?P<date>[A-Z][a-z]* \d{1,2})"
rm = re.search(pattern, description)
if rm is not None:
    date = rm.group("date")
    dt = datetime.strptime(date, "%B %d")
else:
    dt = None
```